### DISCUSSION TITLE
i have lost xch from my wallet

### DISCUSSION THREAD

**Discussion by hirangrg** [Chia-Network/chia-blockchain#10782] [Support]
there was 1.75 xch reward which i earned from self pooling and now its not in my wallet no outgoing transaction, is there anyway i can retrieve it or its lost?

**Comment by Ealrann** [comment]
Hello, which version of Chia do you use? Can you search your address on an explorer to see if you see the same result (like https://xchscan.com/ )?

**Comment by yagmurseyduna** [comment]
The amount of chi in my wallet has suddenly decreased, why?

**Comment by hirangrg** [comment]
Transaction Details
Buy
Exchange
Earn
Txn Hash
0x751ec5ceceebd2def8fc437545c310d0b6e918af8b733a42f9a98338a2f46b83
Status
Success - Spent
Block
[1,640,794](https://xchscan.com/blocks/1640794)
Timestamp
23 days ago (Mar-03-2022 4:46:03 AM) +UTC
Value
1.75 XCH ($119.93)
XCH Price
$76.51 / XCH
From
[xch1rhtjezuqdrh8vur0l5grt07mkss2wqtdgwme3wylxy259x4uav5qdr52h7](https://xchscan.com/address/xch1rhtjezuqdrh8vur0l5grt07mkss2wqtdgwme3wylxy259x4uav5qdr52h7)
To
[xch180anmzl208aaz7rq6s7r2af5ekttw9wgqnqx90j4fv3h5fx60ceqs7qys5](https://xchscan.com/address/xch180anmzl208aaz7rq6s7r2af5ekttw9wgqnqx90j4fv3h5fx60ceqs7qys5)
Type
Transaction

**Comment by hirangrg** [comment]
chia version 1.2.11

**Comment by serge2martel** [comment]
Looks like I have the same problem. With the withdrawal of 1.75 XCH from the unclaimed rewards, almost all of them (1.7498 XCH) were transferred to an address unknown to me. And at the same time there was a payment of 1.75 XCH to my address and at the same height a transfer from this address to an unknown address. As a result, my wallet is missing 1.75
https://xchscan.com/address/xch1s7wfkaghrlkuy342tm94j7q5x7uw39p6gk9skhu4tew8r2dgk6kqfcnrl8

**Comment by hirangrg** [comment]
does anyone knows how to retrieve those lost chia as i have won it during self pooling and it is not in my wallet..

**Comment by Eugene200** [comment]
I have same issue lost 0.6 xch since upgrade from 1.2* to 1.3.1

**Comment by loppefaaret** [comment]
First, make sure you are using 1.3.1
then, you can try and close down chia, and manually remove the current wallet database from `~\.chia\mainnet\wallet\db\`
then open up chia again, and let it resync a fresh wallet database from 0
if you are still having issues, please let me know

**Reply by hirangrg** [reply]
I HAVE INSTALLED NEW CHIA 1.3.1 IT DOES SHOW MY MISSING XCH BUT ON PEDING BALANCE?

**Reply by Eugene200** [reply]
worked for me thanks!!!

**Reply by loppefaaret** [reply]
> I HAVE INSTALLED NEW CHIA 1.3.1 IT DOES SHOW MY MISSING XCH BUT ON PEDING BALANCE?

Does the wallet report itself as being fully synced ?

**Reply by hirangrg** [reply]
YES WALLET IS FULLY SYNCED BUT 1.749999 XCH ARE STILL ON PENDING

**Reply by Margovincent** [reply]
this also solved my issues

**Reply by loppefaaret** [reply]
> YES WALLET IS FULLY SYNCED BUT 1.749999 XCH ARE STILL ON PENDING

it might need more time, before actually getting into a block, cause of the dust being spread by someone on the blockchain.

**Reply by serge2martel** [reply]
Didn't work for me even though I updated everything. Perhaps you need to try to do the same, returning the old version 1.2.11 for a while?
https://xchscan.com/address/xch1s7wfkaghrlkuy342tm94j7q5x7uw39p6gk9skhu4tew8r2dgk6kqfcnrl8 - And I did not make this transaction (And I did not make this transaction, I was just trying to collect the 1.75 reward from the pool), I was just trying to collect the 1.75 reward from the pool

**Reply by tslims99** [reply]
This worked for me. Thanks!! :)

**Comment by Margovincent** [comment]
Hi, I have an issue after I update my gui to 1.3.1
A certain transaction history was missing and I lost all earned chia during that time which is more or less 20++ chia coin. Can anyone help me?

![History](https://user-images.githubusercontent.com/102890439/161411285-1d8d883e-29a6-4451-9091-8f0442013029.jpg)

**Reply by Margovincent** [reply]
solved thanks to @[loppefaaret](https://github.com/loppefaaret)
